utils/transfer/examples/transfer-archive.rs

```diff
@@ -6,8 +6,8 @@ use std::env;
 use std::path::PathBuf;
 use structopt::StructOpt;
 use url::Url;
-use ya_client_model::activity::{FileSet, SetEntry, TransferArgs};
-use ya_transfer::{archive, extract};
+use ya_client_model::activity::{FileSet, SetEntry};
```

Comment:
you can remove this example all together, as it is covered in `transfer` example